src/api.js

```diff
@@ -246,8 +248,8 @@ class API {
     (async () => {
       try {
         this._widgetOpened = true;
-        const { provider, accountId } = options;
-        const url = await this.getConnectUrl({ provider, accountId });
+        const { provider, providers, accountId } = options;
+        const url = await this.getConnectUrl({ provider, providers, accountId });
```

Comment:
@oleg-vysotsky should we allow both `provider` and `providers` to be set? I mean if a `provider` is passed in the providers list would be skipped and `providers` param would be useless?